runtime-modules/content/src/lib.rs

```diff
@@ -2256,6 +2100,59 @@ impl<T: Trait> Module<T> {
         }
     }
 
+    pub fn ensure_valid_init_transactional_status(
+        init_status: &InitTransactionalStatus<T>,
+    ) -> Result<
+        TransactionalStatus<
```

Comment:
It makes sense to introduce an alias for such a complex type.

Reply:
fixed in afcde3072cf9c430da415bf088c3e96196d7f02b